e2e-tests/ukef/cypress/support/portal-api/getOneDeal.js

```diff
@@ -0,0 +1,13 @@
+const { getDeal, logIn } = require('./api');
+
+/**
+ * User get a deal on portal
```

Comment:
```suggestion
 * Get a deal from the portal API
```

As it's not a user getting a deal technically 😄 

Is this for BSS/EWCS and GEF?

Reply:
@ttbarnes yes it is for both and it blocks me the other pull request I have opened